frontend/src/features/modules/module.utils.ts

```diff
@@ -1,4 +1,16 @@
+import {
+  ArrowDownOnSquareStackIcon,
+  BellAlertIcon,
+  BoltIcon,
+  CircleStackIcon,
+  CodeBracketIcon,
+  CogIcon,
+  LockClosedIcon,
+  RectangleGroupIcon,
+} from '@heroicons/react/24/outline'
+import type { ForwardRefExoticComponent, SVGProps } from 'react'
```

Comment:
lol exotic component is such a good name

Reply:
So exotic 😂